src/main/java/frc/robot/subsystems/endgame/EndgameSubsystem.java

```diff
@@ -33,18 +33,22 @@ public EndgameSubsystem(EndgameIO endgameIO) {
     public void setAction(EndgameAction action) {
         switch (action) {
             case GO_UP:
-                endgameIo.setVolts(4.0);
+                endgameIo.setPosition(1.0);
+                endgameIo.setOverrideMode(false);
                 state = State.NORMAL;
                 break;
             case GO_DOWN:
-                endgameIo.setVolts(-4.0);
+                endgameIo.setPosition(0.0);
```

Comment:
We should set this to a constant in Constants.java as well. Even though it is 0 right now, there is justification for making it a negative number and then believing in the current limit